stdlib/float.mli

```diff
@@ -589,60 +597,239 @@ module Array : sig
 
 
   val map_to_array : (float -> 'a) -> t -> 'a array
-  (** [map_to_array f a] applies function [f] to all the elements of [a],
+  (** [map_to_array ~f a] applies function [f] to all the elements of [a],
       and builds an array with the results returned by [f]:
       [[| f a.(0); f a.(1); ...; f a.(length a - 1) |]]. *)
 
   val map_from_array : ('a -> float) -> 'a array -> t
-  (** [map_from_array f a] applies function [f] to all the elements of [a],
+  (** [map_from_array ~f a] applies function [f] to all the elements of [a],
       and builds a floatarray with the results returned by [f]. *)
 
+  (**/**)
```

Comment:
I can't remember if `(**/**)` scopes to the end of the submodule - does this suppress the documentation of `ArrayLabels` below?

Reply:
Yes, I have verified visually by building the manual that it does the right thing.